### ISSUE TITLE
When to support django 2.2

### ISSUE THREAD

**Issue by rayer4u** [noripyt/django-cachalot#130]
Thanks for this project, I'am using it to simplify table cache. But when I using in django 2.2, error happened
```
ERRORS:
?: (cachalot.E003) Django 2.2.1 is not compatible with this version of django-cachalot.
        HINT: Refer to the django-cachalot documentation to find which versions are compatible.
```
Is there any plan to support django 2.2? I think django 2.2 does not change a lot, the upgrade should be easy.

**Comment by editkid** [comment]
This'd be especially nice since Django 2.2 is designated as a long-term support (LTS) release.

**Comment by editkid** [comment]
Looks like it's in progress: https://github.com/noripyt/django-cachalot/pull/127

**Comment by rayer4u** [comment]
> Looks like it's in progress: #127

Thanks a lot. I'm suppressing  the version check just for now.
```settings.py
SILENCED_SYSTEM_CHECKS = ["cachalot.E003"]
```

**Comment by fabiocaccamo** [comment]
When will you release a new version with `django==2.2` support?

**Comment by fabiocaccamo** [comment]
Why you closed this issue? `django==2.2` is not supported yet by any release on pypi.

**Comment by Andrew-Chen-Wang** [comment]
@rayer4u @fabiocaccamo Package released for Django 2.2 and 3.0 support.

**Comment by fabiocaccamo** [comment]
Great work @Andrew-Chen-Wang !
Thank you very much!!
